test/unit/foreman/renderer/snapshots/ProvisioningTemplate/provision/XenServer default answerfile.snap.txt

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0"?>
 <installation mode="fresh" srtype="lvm">
-zerombr\nclearpart --all    --initlabel\npart /boot --fstype ext3 --size=100 --asprimary\npart /     --f   stype ext3 --size=1024 --grow\npart swap  --recommended  <keymap>us</keymap>
+zerombr
+clearpart --all    --initlabel
+part /boot --fstype ext3 --size=100 --asprimary
+part /     --fstype ext3 --size=1024 --grow
+part swap  --recommended  <keymap>us</keymap>
```

Comment:
Just reading this, I highly doubt this is correct. Doesn't look like valid XML. Maybe something for another day.

It does look like we need the correct OS to be set for this.

Reply:
I am fixing the newline, but definitely this does not look good. This is probably broken for some time I guess, I do not use Xen and I am not planning to :)

Reply:
Ah I think I know. This is an incorrect snapshot test, I am removing it from `snapshots.yaml` because in snapshots we only test with EL7 and this obviously needs a different partitioning template.

There is a PR where I extend snapshot tests to other OSes or scenarios, XenServer however is not one of them :)